src/tester.rs

```diff
@@ -661,6 +667,8 @@ fn run_cmd(
     let mut stderr = FileDescriptor::dup(child.stderr.as_ref().unwrap()).unwrap();
     let mut stdout = FileDescriptor::dup(child.stdout.as_ref().unwrap()).unwrap();
 
+    non_blocking(&stderr, &stdout).unwrap_or_else(|_| fatal("Couldn't perform non-blocking."));
```

Comment:
Let's change this message to `Couldn't set stderr and stdout to be non-blocking`.